### PR TITLE
GINI IOSP improvements

### PR DESCRIPTION
**Pull request by lesserwhirls**
Add new info based on http://www.nws.noaa.gov/noaaport/document/ICD%20CH5-2005-1.pdf

Fix incorrect Unidata specific product info based on https://github.com/Unidata/gempak/blob/master/gempak/tables/unidata/nex2gini.tbl

### CODE REVIEW

File: clcommon/src/main/java/ucar/nc2/iosp/gini/Giniheader.java [hunk 1]
```diff
@@ -751,30 +716,42 @@ String gini_GetEntityID(int ent_id) {
...
+      case 17:
+        name = "GOES-14 satellite Image";
+        break;
+      case 18:
+        name = "GOES-15 satellite Image";
+        break;
+      case 19: // GOES-R
+        name = "GOES-16 satellite Image";
+        break;
+      case 99: // special snowflake GEMPAK Composite Images generated by Unidata
```
Comment (dopplershift) [inline clcommon/src/main/java/ucar/nc2/iosp/gini/Giniheader.java]: 😭

**Comment by dopplershift** [comment]
I hope you didn't search hard for that ICD, since I have a copy.

**Comment by lesserwhirls** [comment]
Nah, it was easy to find thanks to talking to you and knowing what to look for.

**Comment by cwardgar** [comment]
I wish we had tests for Gini, but looking at the IOSP, that's a big job.

**Pull request merged**